src/list.js

```diff
@@ -293,6 +293,11 @@ Sk.builtin.list.prototype.sq$contains = function (item) {
     }
     return false;
 };
+
+Sk.builtin.list.prototype.__contains__ = function(self, item) {
```

Comment:
Right you are, both times. I feel like I owe you a drink for each stupid mistake of mine you spot in code review, but at this rate that would get messy quickly...